zebra-chain/src/transaction/serialize.rs

```diff
@@ -335,7 +364,67 @@ impl ZcashDeserialize for Transaction {
                 let inputs = Vec::zcash_deserialize(&mut reader)?;
                 let outputs = Vec::zcash_deserialize(&mut reader)?;
 
```

Comment:
Please follow the field order in the v5 transaction spec.

<img width="827" alt="Screen Shot 2021-04-15 at 09 51 18" src="https://user-images.githubusercontent.com/8951843/114796893-55528b80-9dd5-11eb-9763-9d0713bd37d9.png">